common/source/docs/common-advanced-configuration.rst

```diff
@@ -63,6 +63,7 @@ tuning options for the vehicle.
     Notch Filter Configuration <common-imu-notch-filtering>
 [/site]
     Object Avoidance <common-object-avoidance-landing-page>
+    Operation Using Only a GCS <common-gcs-only-operation>
```

Comment:
Alternative names to consider would be "GCS only flying" or "Flying without an RC".  The existing one isn't bad though just food for thought.

Reply:
changed to Ground Control Station Only Operation